recipes/launchd_service.rb

```diff
@@ -15,19 +15,38 @@ class ::Chef::Recipe
 mac_service_supported = version_checker.satisfied_by?(Gem::Version.new(node['chef_packages']['chef']['version']))
 
 if mac_service_supported
-  template '/Library/LaunchDaemons/com.chef.chef-client.plist' do
-    source 'com.chef.chef-client.plist.erb'
-    mode 0644
-    variables(
-      :launchd_mode => node['chef_client']['launchd_mode'],
-      :client_bin => client_bin
-    )
+  if node['chef_client']['task']['user'].eql? 'SYSTEM'
```

Comment:
The `node['chef_client']['task']['user']` attribute is for Windows, specifically for managing the Chef Client as a scheduled task (Windows' cron-alike).